# Waterfox for Android and iOS isn't available, so remove that
diff --git a/browser/components/preferences/sync.inc.xhtml b/browser/components/preferences/sync.inc.xhtml
index 310e83e8ba3b..66ac79c41602 100644
--- a/browser/components/preferences/sync.inc.xhtml
+++ b/browser/components/preferences/sync.inc.xhtml
@@ -35,22 +35,6 @@
         </hbox>
       </vbox>
     </hbox>
-    <label class="fxaMobilePromo" data-l10n-id="sync-mobile-promo">
-      <html:img
-        src="chrome://browser/skin/logo-android.svg"
-        data-l10n-name="android-icon"
-        class="androidIcon"/>
-      <html:a
-        data-l10n-name="android-link"
-        class="fxaMobilePromo-android text-link" target="_blank"/>
-      <html:img
-        src="chrome://browser/skin/logo-ios.svg"
-        data-l10n-name="ios-icon"
-        class="iOSIcon"/>
-      <html:a
-        data-l10n-name="ios-link"
-        class="fxaMobilePromo-ios text-link" target="_blank"/>
-    </label>
   </groupbox>
 
   <vbox id="hasFxaAccount">
diff --git a/browser/components/preferences/sync.js b/browser/components/preferences/sync.js
index 6418e4dbec9f..7c179028e9e9 100644
--- a/browser/components/preferences/sync.js
+++ b/browser/components/preferences/sync.js
@@ -121,17 +121,6 @@ var gSyncPane = {
           .getElementById("connect-another-device")
           .setAttribute("href", connectURI);
       });
-    // Links for mobile devices.
-    for (let platform of ["android", "ios"]) {
-      let url =
-        Services.prefs.getCharPref(`identity.mobilepromo.${platform}`) +
-        "sync-preferences";
-      for (let elt of document.querySelectorAll(
-        `.fxaMobilePromo-${platform}`
-      )) {
-        elt.setAttribute("href", url);
-      }
-    }
 
     this.updateWeavePrefs();
 
